packages/cacti-copm-core/src/main/json/openapi.json

```diff
@@ -301,6 +286,24 @@
     }
   },
   "paths": {
+    "/api/v1/plugins/@hyperledger/cacti-plugin-copm/no-op": {
+      "post": {
+        "x-hyperledger-cacti": {
+          "http": {
+            "path": "/api/v1/plugins/@hyperledger/cacti-plugin-copm/no-op",
```

Comment:
Change `@hyperledger` --> `@hyperledger-cacti` here and in other locations.

Reply:
Done